packages/riipen-ui/src/components/AccordionDetails.jsx

```diff
@@ -0,0 +1,36 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+const AccordionDetails = props => {
+  const { classes, ...other } = props;
+
+  return (
+    <React.Fragment>
+      <div className={clsx("root", classes)} {...other} />
+      <style jsx>{`
+        .root {
+          display: flex;
+        }
+      `}</style>
+    </React.Fragment>
+  );
+};
+
+AccordionDetails.propTypes = {
+  /**
+   * The content of the accordion details.
+   */
+  children: PropTypes.node,
+
+  /**
+   * List of additional classes to apply to this component.
+   */
+  classes: PropTypes.array
+};
+
+AccordionDetails.defaultProps = {};
+
+AccordionDetails.displayName = "AccordionDetails";
```

Comment:
and do we need to set this? should get set to the variable name

Reply:
Ya we have to set this on all components. This allows us to do stuff like `:global(.riipen).whatever` in our CSS on `platform-web` and have it actually work.

Reply:
ooooh right, i remember that css bug now. ok sweet